057-AzureRouteServer/Coach/Solution-03.md

```diff
@@ -0,0 +1,20 @@
+# Challenge 03 - <Title of Challenge> - Coach's Guide 
```

Comment:
Is there no coach guide for challenge 3?

Reply:
@ravibak, any update? @perktime in order to add the coaching guide can we do it on my fork?